src/lib/messenger/fullwidth.ts

```diff
@@ -0,0 +1,44 @@
+import { isObject } from '@guardian/libs';
+import fastdom from 'fastdom';
+import type { RegisterListener } from 'core/messenger';
+
+interface FullwidthSpecs {
+	fullwidth: boolean;
+}
+
+const isFullwidthSpecs = (specs: unknown): specs is FullwidthSpecs =>
+	!!specs && isObject(specs) && 'fullwidth' in specs;
```

Comment:
guardian/libs has got a `isBoolean` function if you wanted to validate the type of the `fullwidth` property as well